src/select/select.js

```diff
@@ -256,7 +261,12 @@ class Select extends React.Component<PropsT, StatelessStateT> {
           InputContainer: InputContainer,
           After: () => (
             <SearchIcon
-              onClick={e => this.onChange(e, STATE_CHANGE_TYPE.clearAll)}
+              disabled={disabled}
```

Comment:
this attribute will leak to DOM <img> tag, but it's not supported by this DOM element, that's why we name them `$disabled` if we want it to be ignored

Reply:
Thanks for the explanation @blackswanny!

Reply:
There should be a set of shared props that are passed to all subelements, I'd say the `disabled` props should be one of them, all props in the shared set should be prefixed with `$` sign